### DISCUSSION TITLE
500 error when getting manifests

### DISCUSSION THREAD

**Discussion by notsle** [Homebrew/discussions#4156] [Everyday usage] [answered]
### Output of `brew config`

```shell
➜  ~ brew config
HOMEBREW_VERSION: 3.6.20
ORIGIN: https://github.com/Homebrew/brew
HEAD: 344d32bf7f40183359592ef81ca6486e0587981b
Last commit: 12 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 369d38a2dd04607af4199bd3cc87ef5db86fb881
Core tap last commit: 3 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.39.0 => /opt/homebrew/bin/git
Curl: 7.85.0 => /usr/bin/curl
macOS: 13.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
Rosetta 2: false
```

### Output of `brew doctor`

```shell
➜  ~ brew doctor
Your system is ready to brew.
```

### Description of issue

Attempting to install a new formula or upgrade existing gets 500 error when pulling down manifest 
==> Fetching fontconfig
==> Downloading https://ghcr.io/v2/homebrew/core/fontconfig/manifests/2.14.2
curl: (22) The requested URL returned error: 500                              
Warning: Problem : HTTP error. Will retry in 1 seconds. 3 retries left.
curl: (22) The requested URL returned error: 500                              
Warning: Problem : HTTP error. Will retry in 2 seconds. 2 retries left.
curl: (22) The requested URL returned error: 500                              
Warning: Problem : HTTP error. Will retry in 4 seconds. 1 retries left.
curl: (22) The requested URL returned error: 500                              

Error: fontconfig: Failed to download resource "fontconfig_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/fontconfig/manifests/2.14.2

**Comment by Aditya-Peshin** [comment, marked as answer]
Looks like it's a github issue. The status page shows degraded performance for packages - https://www.githubstatus.com/

**Reply by FlavioB79** [reply]
> Looks like it's a github issue. The status page shows degraded performance for packages - https://www.githubstatus.com/

Uh, thanks. I'm suffering the same issue.

**Reply by Aditya-Peshin** [reply]
You can follow the incident here: https://www.githubstatus.com/incidents/zf2tk6x40gmh

**Reply by notsle** [reply]
Of course when i checked githubstatus it was before the issue was acknowledged ;-) Thanks.

**Reply by Aditya-Peshin** [reply]
Incident resolved, I've got brew working for me normally now.
